Testscripts/Linux/utils.sh

```diff
@@ -1387,25 +1362,24 @@ CreateIfupConfigFile()
 			debian*|ubuntu*)
 				__file_path="/etc/network/interfaces"
 				if [ ! -d "$(dirname $__file_path)" ]; then
-					LogMsg "CreateIfupConfigFile: $(dirname $__file_path) does not exist! Something is wrong with the network config!"
-					return 3
+					LogErr "$(dirname $__file_path) does not exist! Something is wrong with the network config!"
+					return 1
 				fi
 
 				if [ -e "$__file_path" ]; then
-					LogMsg "CreateIfupConfigFile: Warning will overwrite $__file_path ."
+					LogErr "Warning will overwrite $__file_path ."
 				fi
 
 				#Check if interface is already configured. If so, delete old config
-				if grep -q "$__interface_name" $__file_path
-				then
-					LogMsg "CreateIfupConfigFile: Warning will delete older configuration of interface $__interface_name"
-				    lineNumber=$(cat -n $__file_path |grep "iface $__interface_name"| awk '{print $1;}')
-				    if [ $lineNumber ]; then
-				        lineNumber=$lineNumber+1
-				        sed -i "${lineNumber},+1 d" $__file_path
-				    fi
-				    sed -i "/$__interface_name/d" $__file_path
-				fi
+				if [ grep -q "$__interface_name" $__file_path ]; then
+                    LogErr "Warning will delete older configuration of interface $__interface_name"
+                    lineNumber=$(cat -n $__file_path |grep "iface $__interface_name"| awk '{print $1;}')
+                    if [ $lineNumber ]; then
+                        lineNumber=$lineNumber+1
+                        sed -i "${lineNumber},+1 d" $__file_path
+                    fi
+                    sed -i "/$__interface_name/d" $__file_path
+                fi
```

Comment:
indent is inconsistent between 1374 - 1382

Reply:
fixed